### PR TITLE
fix: Added missing Norwegian translations

### LINKED ISSUE #1707: Help improve Appwrite's Norwegian language translation ✍️
# Introduction

Did you know Appwrite's Locale API has support for returning responses in your own locale based on the value of the `X-Appwrite-Locale` header? 😮 We already support 56 languages and need your help to improve them! 💪

Your task is to update Appwrite's **Norwegian** language translations. We recently added new features, so some translations might be missing. You can find a complete in-depth tutorial in our official [tutorial for adding translations](https://github.com/appwrite/appwrite/blob/master/docs/tutorials/add-translations.md).

This is what missing translation would look like:

```json
"emails.magicSession.footer": "",
```

### Tasks summary:

- [ ] Fork & clone Appwrite
- [ ] Update translations for Norwegian in `app/config/locale/translations/no.json`
- [ ] Test the translations
- [ ] Push changes and submit pull request

If you need any help, reach out to us on our [Discord server](https://discord.gg/GSeTUeA).

Are you ready to work on this issue? 🤔 Let us know, and we will assign it to you 😊

**Happy Appwriting!**

---

### PR DESCRIPTION
**Pull request by NeonSpork**
## What does this PR do?

This pull request adds the missing Norwegian translations in `app/config/locale/translations/no.json`

## Test Plan

I tested the changes following the instructions in [add-translations.md](https://github.com/appwrite/appwrite/blob/master/docs/tutorials/add-translations.md), and received a `200 OK` request response.

## Related PRs and Issues

This resolves #1707 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes I have, and followed the guidelines to the best of my ability.

### CODE REVIEW

File: app/config/locale/translations/no.json [hunk 1]
```diff
@@ -3,30 +3,30 @@
...
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Kontobekreftelse",
+    "emails.verification.hello": "Hallo {{name}}",
+    "emails.verification.body": "Følg denne lengken for å bekrefte din e-post adresse",
+    "emails.verification.footer": "Dersom du ikke ba om å bekrefte e-post adressen, kan du se bort fra denne meldingen.",
+    "emails.verification.thanks": "Takk",
+    "emails.verification.signature": "{{project}} team",
+    "emails.magicSession.subject": "Pålogging",
+    "emails.magicSession.hello": "Hallo",
```
Comment (christyjacob4) [inline app/config/locale/translations/no.json:13]:
```suggestion
    "emails.magicSession.hello": "Hallo,",
```
Please follow the punctuation from `en.json`

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

File: app/config/locale/translations/no.json (hunk 1 above)
Comment (NeonSpork) [reply app/config/locale/translations/no.json:13]: Oops, small typo, I'll fix that right away!

Comment (NeonSpork) [reply app/config/locale/translations/no.json]: I changed it to "Hei," which more closely follows the equivalent line in `en.json`

Comment (christyjacob4) [reply app/config/locale/translations/no.json]: You can resolve it yourself if you feel the issue has been addressed

Comment (christyjacob4) [reply app/config/locale/translations/no.json]: Also please check the punctuation in all the other string as well. Not just line 13

Comment (NeonSpork) [reply app/config/locale/translations/no.json]: Went over everything and found another period that had eluded me. Also fixed a typo. The changes I made follow the same punctuation as in the English version as far as I can see, so I'm resolving it now. :)

**Comment by NeonSpork** [comment]
Not quite sure why the Travis CI fails, it passed the first time, and I'm not so sure it's directly related to my .json changes?

**Comment by christyjacob4** [comment]
THANK YOU so much! All changes merged 🥳
Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swag!

**Pull request merged**